lib/peeps.rb

```diff
@@ -0,0 +1,48 @@
+require 'pg'
+
+class Peeps
+  
+  attr_reader :id, :date, :message
+
+  def initialize(id:, date:, message:)
+    @id = id
+    @date = date
+    @message = message
+  end
+
+  def self.all
+
```

Comment:
This is good, in that it works, but I'd probably move the logic that determines which db is used into another class.